ichnaea/webapp/app.py

```diff
@@ -51,20 +49,37 @@ def worker_exit(server, worker):
     shutdown_worker(_APP)
 
 
-def log_access_factory(wsgi_app):
-    """WSGI middleware for logging HTTP requests."""
+def log_middleware(wsgi_app):
+    """WSGI middleware for logging."""
 
     def handle(environ, start_response):
         method = environ["REQUEST_METHOD"]
         path = environ.get("PATH_INFO", "")
+        start = time.time()
```

Comment:
This seemed like a better place to time requests to me. This means that the logs will have slightly longer times than the metrics version from `log_tween_factory`.